### PR TITLE
Forum channels

### PR DESCRIPTION
**Pull request by jaw0r3k**
- [x] Changes guide
- [x] FaQ about creating posts
> Yeah i know it will be merged when 14.4.0 is released

### CODE REVIEW

**Comment by jaw0r3k** [comment]
And feel free to say your suggestions

**Comment by SpokieKid** [comment]
has this been released to the website by now? Can't seem to find relevant materials towards Forum

**Comment by Jiralite** [comment]
Of course not - this is still a pull request.

File: guide/additional-info/changes-in-v14.md [hunk 1]
```diff
@@ -531,6 +532,16 @@ A new `ignore` event has been added which is emitted whenever an element is not
...
 
+### ForumChannel
+
+Added class `ForumChannel` which represents GuildForum.
+
+# GuildForumThreadManager 
+Added `GuildForumThreadManager` as manager for threads in forum channels.
+
+# GuildTextThreadManager
+Added `GuildTextThreadManager` as manager for threads in text channels.
```
Comment (Jiralite) [inline guide/additional-info/changes-in-v14.md:543]: These are not in alphabetical order.

**Comment by Nex2944** [comment]
> has this been released to the website by now? Can't seem to find relevant materials towards Forum
https://deploy-preview-1265--discordjs-guide.netlify.app/additional-info/changes-in-v14.html#forumchannel works

File: guide/additional-info/changes-in-v14.md [hunk 2]
```diff
@@ -531,6 +532,16 @@ A new `ignore` event has been added which is emitted whenever an element is not
 
 `CommandInteraction#commandGuildId` has been added which is the id of the guild the invoked application command is registered to.
 
+### ForumChannel
+
+Added class `ForumChannel` which represents GuildForum.
```
Comment (Jiralite) [inline guide/additional-info/changes-in-v14.md:537]:
```suggestion
discord.js supports forum channels as of 14.4.0.
```

File: guide/additional-info/changes-in-v14.md [hunk 3]
```diff
@@ -551,10 +562,14 @@ Added `GuildEmojiManager#delete()` and `GuildEmojiManager#edit()` for managing e
 
 Added `Interaction#isRepliable()` to check whether a given interaction can be replied to.
 
+### Message
+`Message#position` has been added as approximate position in a thread.
```
Comment (Jiralite) [inline guide/additional-info/changes-in-v14.md:566]:
```suggestion
### Message

`Message#position` has been added as an approximate position in a thread as of 14.4.0.
```

File: guide/additional-info/changes-in-v14.md [hunk 4]
```diff
@@ -551,10 +562,14 @@ Added `GuildEmojiManager#delete()` and `GuildEmojiManager#edit()` for managing e
...
+
 ### MessageReaction
 
 Added `MessageReaction#react()` to make the client user react with the reaction the class belongs to.
 
 ### Webhook
 
 Added `Webhook#applicationId`.
+Added `Webhook#threadName`.
```
Comment (Jiralite) [inline guide/additional-info/changes-in-v14.md:575]:
```suggestion

Added `Webhook#threadName` as of 14.4.0 which allows a webhook to create a post in a forum channel.
```

File: guide/additional-info/changes-in-v14.md [hunk 5]
```diff
@@ -507,6 +507,7 @@ Many of the analogous enums can be found in the discord-api-types docs. [discord
 `AutocompleteInteraction#commandGuildId` has been added which is the id of the guild the invoked application command is registered to.
 
 ### Channel
+`Channel#flags` has been added.
```
Comment (Jiralite) [inline guide/additional-info/changes-in-v14.md:510]:
```suggestion
### Channel

`Channel#flags` has been added as of 14.4.0.
```

**Review by Jiralite — CHANGES_REQUESTED** [review]

File: guide/additional-info/changes-in-v14.md (hunk 2 above)
Comment (jaw0r3k) [reply guide/additional-info/changes-in-v14.md]: Maybe for me 
But it looks little strange in guide 
image

Comment (jaw0r3k) [reply guide/additional-info/changes-in-v14.md]: `Added support for forum channels as of 14.4.0.` sounds better?
`

Comment (Jiralite) [reply guide/additional-info/changes-in-v14.md]: Sure.

File: guide/additional-info/changes-in-v14.md (hunk 1 above)
Comment (Jiralite) [reply guide/additional-info/changes-in-v14.md]: They should be in alphabetical order. It should be `GuildChannelManager` -> `GuildEmojiManager ` -> `GuildForumThreadManager` -> `GuildMemberManager` -> `GuildTextThreadManager`. You can correct the order too!

Comment (jaw0r3k) [reply guide/additional-info/changes-in-v14.md]: K

File: guide/additional-info/changes-in-v14.md [hunk 6]
```diff
@@ -539,22 +545,36 @@ Added `Guild#setMFALevel()` which sets the guild's MFA level.
...
+
 ### MessageReaction
 
 Added `MessageReaction#react()` to make the client user react with the reaction the class belongs to.
 
 ### Webhook
 
 Added `Webhook#applicationId`.
+
+Added `Webhook#threadName` as of 14.4.0 which allows a webhook to create a post in a forum channel.
```
Comment (almostSouji) [inline guide/additional-info/changes-in-v14.md:580]: Not on Webhook, but when sending a webhook in the options.

**Review by almostSouji — CHANGES_REQUESTED** [review]

File: guide/additional-info/changes-in-v14.md (hunk 6 above)
Comment (jaw0r3k) [reply guide/additional-info/changes-in-v14.md:580]: Ah yeah

Comment (almostSouji) [reply guide/additional-info/changes-in-v14.md:562]: Well, mention that a new option exists / demonstrate it with a minimal use sample

File: guide/additional-info/changes-in-v14.md [hunk 7]
```diff
@@ -520,24 +526,37 @@ Added `Guild#maxVideoChannelUsers` as of 14.2.0 which indicates the maximum numb
...
```
Comment (jaw0r3k) [inline guide/additional-info/changes-in-v14.md:562]:
```suggestion
Added `threadName` property in `Webhook#send()` options as of 14.4.0 which allows a webhook to create a post in a forum channel.
```
Is it good?

Comment (jaw0r3k) [reply guide/additional-info/changes-in-v14.md:562]: @almostSouji ?

Comment (almostSouji) [reply guide/additional-info/changes-in-v14.md:562]: Added property X in Y options...

Comment (jaw0r3k) [reply guide/additional-info/changes-in-v14.md:562]: I can't find anything more creative

Comment (almostSouji) [reply guide/additional-info/changes-in-v14.md:562]: Unsure what you mean by creative? use what you suggested and apply the order changes I requested above and it's fine

**Review by almostSouji — APPROVED** [review]

File: guide/popular-topics/faq.md [hunk 8]
```diff
@@ -118,6 +118,16 @@ client.user.setPresence({ activities: [{ name: 'activity' }], status: 'idle' });
 const channel = client.channels.cache.get('id');
 channel.send('content');
 ```
+### How do I create a post in a forum channel?
```
Comment (almostSouji) [inline guide/popular-topics/faq.md:121]: heading needs an additional empty line before it

**Review by almostSouji — CHANGES_REQUESTED** [review]

**Review by almostSouji — APPROVED** [review]

**Pull request merged**